### ISSUE TITLE
The miou is pretty low using pspnet on cityscapes

### ISSUE THREAD

**Issue by wkh000** [yassouali/pytorch_segmentation#104]
I was trying to train a pspnet on cityscapes dataset, and the miou only achieve about 60% on the val dataset, can you give me your config for training a pspnet model on the dataset, thanks a lot

**Comment by wkh000** [comment]
{
    "name": "PSPNet",
    "n_gpu": 2,
    "use_synch_bn": true,

    "arch": {
        "type": "PSPNet",
        "args": {
            "backbone": "resnet101",
            "freeze_bn": true,
            "freeze_backbone": false
        }
    },

    "train_loader": {
        "type": "CityScapes",
        "args":{
            "data_dir": "Cityscape",
            "batch_size":14,
            "base_size": 512,
            "crop_size": 353,
            "augment": true,
            "shuffle": true,
            "scale": true,
            "flip": true,
            "rotate": true,
            "blur": false,
            "split": "train",
            "num_workers": 8
        }
    },

    "val_loader": {
        "type": "CityScapes",
        "args":{
            "data_dir": "Cityscape",
            "batch_size": 4,
            "crop_size": 1024,
            "val": true,
            "split": "val",
            "num_workers": 6
        }
    },

    "optimizer": {
        "type": "SGD",
        "differential_lr": true,
        "args":{
            "lr": 0.01,
            "weight_decay": 1e-4,
            "momentum": 0.9
        }
    },

    "loss": "CrossEntropyLoss2d",
    "ignore_index": 255,
    "lr_scheduler": {
        "type": "Poly",
        "args": {}
    },

    "trainer": {
        "epochs": 100,
        "save_dir": "saved/",
        "save_period": 10,
  
        "monitor": "max Mean_IoU",
        "early_stop": 10,
        
        "tensorboard": true,
        "log_dir": "saved/runs",
        "log_per_iter": 20,

        "val": true,
        "val_per_epochs": 2
    }
}

**Comment by wkh000** [comment]
here is my config, to save the memory I set the base_size in train_loader as 512, but the trained model's miou is only 59%, do you have any advice? I'm a beginner of semantic segmentation.

**Comment by yassouali** [comment]
@wkh000 For cityscapes, you'll need to train for a bit longer and used larger images (1024 for example)
